packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -59,15 +61,16 @@ export default {
             if (isOpen && this.link.imActive && this.link.path) {
                 this.$el.scrollIntoView({block: 'center'});
             }
-        }
+        },
     },
     methods: {
-        toggle() {
-            this.iHaveChildrenActive = !this.iHaveChildrenActive
+        toggleExpanded() {
+            this.sublinksExpanded = !this.sublinksExpanded
         },
-        setData: function() {
-            this.iHaveChildrenActive = Boolean(this.link.imActive);
+        setData(){
+          this.iHaveChildrenActive = Boolean(this.link.iHaveChildrenActive);
```

Comment:
it looks like we can replace local `iHaveChildrenActive` property with `sublinksExpaned` in watchers section
and use props value (link.iHaveChildrenActive) in template checks